### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by simleo**
* Make the code PEP8-compliant
* Activate flake8 in Travis
* Other minor cleanup actions

### CODE REVIEW

File: biotracks/plot.py [hunk 1]
```diff
@@ -34,21 +34,6 @@
...
-    x -- the x_coordinate header
-    y -- the y_coordinate header
-    t -- the time header
-    """
-    x_coord = x
-    y_coord = y
-    time = t
-    df.sort_values(time, axis=0, inplace=True)
-
-
```
Comment (pcmasuzzo) [inline biotracks/plot.py:51]: I honestly do not recall why I added this function.
No longer needed?

File: scripts/create_dpkg.py [hunk 2]
```diff
@@ -134,7 +134,7 @@ def main(argv):
     y = track_dict.get(names.Y_COORD_NAME)
     frame = track_dict.get(names.FRAME_NAME)
     # basic visualizations
-    plot.prepareforplot(objects_links_tracks, x, y, frame)
+    objects_links_tracks.sort_values(frame, axis=0, inplace=True)
```
Comment (pcmasuzzo) [inline scripts/create_dpkg.py]: OK I see this is just sorting the time frames here.
Good.

**Review by pcmasuzzo — APPROVED** [review]
Good to me

File: biotracks/plot.py (hunk 1 above)
Comment (simleo) [reply biotracks/plot.py:51]: It's a one-liner (the three assignments are unnecessary) and it's used only once, so I've just replaced that single function call with the sort itself.

Comment (pcmasuzzo) [reply biotracks/plot.py:51]: 👍

**Pull request merged**